### PR TITLE
feat: support blobs that are ADR74 emotes

### PR DESCRIPTION
**Pull request by cazala**
~This PR depends on this one to be released before merging: https://github.com/decentraland/ui/pull/199~ Ready ✅

### CODE REVIEW

File: src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx [hunk 1]
```diff
@@ -603,7 +603,7 @@ export default class CreateSingleItemModal extends React.PureComponent<Props, St
   renderWearablePreview = () => {
     const { type, contents } = this.state
     const isEmote = type === ItemType.EMOTE
-    const blob = contents ? toWearableWithBlobs({ contents, isEmote }) : undefined
+    const blob = contents ? toEmoteWithBlobs({ contents }) : undefined
```
Comment (cyaiox) [inline src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx:606]: the method `renderWearablePreview` also computes the thumbnail for wearables. 
the blob should be something like this:
`const blob = contents ? (isEmote ? toEmoteWithBlobs({ contents }) : toWearableWithBlobs({ contents })) : undefined`

Comment (cazala) [reply src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx]: good catch!

**Review by cyaiox — APPROVED** [review]
🚀

**Comment by cazala** [comment]
@cyaiox just a heads up I added one more commit to remove an unnecesary part of the code that checked for a main image file on an emote file, since emotes can't be images.

File: src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx [hunk 2]
```diff
@@ -945,7 +945,7 @@ export default class CreateSingleItemModal extends React.PureComponent<Props, St
     return (
       <EditThumbnailStep
         isLoading={!!isLoading}
-        blob={contents ? toWearableWithBlobs({ contents, isEmote: true }) : undefined}
+        blob={contents ? toEmoteWithBlobs({ contents }) : undefined}
```
Comment (juanmahidalgo) [inline src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx:948]: This should be 
```suggestion
        blob={blob}
```
right?

File: src/components/Modals/EditThumbnailModal/EditThumbnailModal.tsx [hunk 3]
```diff
@@ -61,7 +61,7 @@ export default class EditThumbnailModal extends React.PureComponent<Props, State
 
     let wearablePreviewProp
     if (file) {
-      wearablePreviewProp = { blob: toWearableWithBlobs({ file, isEmote: true }) }
+      wearablePreviewProp = { blob: toEmoteWithBlobs({ file }) }
```
Comment (juanmahidalgo) [inline src/components/Modals/EditThumbnailModal/EditThumbnailModal.tsx:64]: we should add some logic here to know if it should use `toEmoteWithBlobs` or `toWearableWithBlobs`, right?

File: src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx (hunk 2 above)
Comment (cazala) [reply src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx:948]: There’s no ‘blob’ variable in that scope 👀

File: src/components/Modals/EditThumbnailModal/EditThumbnailModal.tsx (hunk 3 above)
Comment (cazala) [reply src/components/Modals/EditThumbnailModal/EditThumbnailModal.tsx:64]: The EditThumbnailModal is just for emotes I think (it is the one with the slider to pick the frame), so no support for WearableWithBlobs

File: src/components/Modals/CreateSingleItemModal/EditThumbnailStep/EditThumbnailStep.types.ts [hunk 4]
```diff
@@ -18,7 +18,7 @@ export enum ControlOptionAction {
 export type Props = {
   title: string
   isLoading: boolean
-  blob?: WearableWithBlobs
+  blob?: EmoteWithBlobs
```
[the remark this answers is not in the document]
Comment (cazala) [reply src/components/Modals/CreateSingleItemModal/EditThumbnailStep/EditThumbnailStep.types.ts:21]: Same answer as above

File: src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx (hunk 2 above)
Comment (juanmahidalgo) [reply src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx:948]: sorry, GitHub trolled me 😛 
image
Now that I'm seeing it correctly, we only need the `toEmoteWithBlobs` here cause it's only for emotes 👍

File: src/components/Modals/EditThumbnailModal/EditThumbnailModal.tsx (hunk 3 above)
Comment (juanmahidalgo) [reply src/components/Modals/EditThumbnailModal/EditThumbnailModal.tsx:64]: Yes, correct! perfect then 👍

**Review by juanmahidalgo — APPROVED** [review]

File: src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx (hunk 2 above)
Comment (cyaiox) [reply src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx:948]: hey @juanmahidalgo!  the method renderWearablePreview is used in the import file step to generate the initial thumbnail for emotes and wearables.

**Pull request merged**